### ISSUE TITLE
Is it possible to "exclude" a line/block

### ISSUE THREAD

**Issue by Ilyes512** [css/csso#236]
Is it possible to ignore a line/block?

I need this cause of the fallowing in bootstrap ([link to the file](https://github.com/twbs/bootstrap-sass/blob/d64d48fd23c9624cc5c996e6516b7dc33fde5919/assets/stylesheets/bootstrap/_dropdowns.scss#L13-L14)):
```scss
// File: bootstrap-sass/assets/stylesheets/bootstrap/_dropdowns.scss

// Dropdown arrow/caret
.caret {
  display: inline-block;
  width: 0;
  height: 0;
  margin-left: 2px;
  vertical-align: middle;
  border-top:   $caret-width-base dashed;
  border-top:   $caret-width-base solid \9; // IE8
  border-right: $caret-width-base solid transparent;
  border-left:  $caret-width-base solid transparent;
}
```

So both border-top need to stay in the output.

**Comment by Ilyes512** [comment]
So... anyone know how to do this? Or else a nice thing to build in (new feature?) by adding something like `/* $csso: ignore */`?

**Comment by stenmuchow** [comment]
+1

**Comment by lahmatiy** [comment]
Fixed by #166
Coming in next release
